internal/liqonet/route-operator.go

```diff
@@ -120,13 +128,18 @@ func (r *RouteController) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		}
 	} else {
 		//the object is being deleted
+		//if we encounter an error while removing iptables rules or the routes than we record an
+		//event on the resource to notify the user
+		//the finalizer is not removed
 		if liqonetOperator.ContainsString(endpoint.Finalizers, routeOperatorFinalizer) {
 			if err := r.deleteIPTablesRulespecForRemoteCluster(&endpoint); err != nil {
 				r.Log.Error(err, "error while deleting rulespec from iptables")
+				r.Recorder.Event(&endpoint, "Error", "", err.Error())
```

Comment:
just a minor thing: can we use "Error" as event type? in the documentation it says only "Normal" and "Warning"

Reply:
you are right. corrected.